### PR TITLE
Change RandomCropper to CenterCrop in ResNet50 preprocessor

### PR DESCRIPTION
**Pull request by hkvision**
- Run evaluation on ImageNet val data and got same results.
Top1Accuracy is Accuracy(correct: 37912, count: 50000, accuracy: 0.75824)
Top5Accuracy is Accuracy(correct: 46332, count: 50000, accuracy: 0.92664)
- Latency for one image preprocessing is around ~2.3 ms and the overall end2end latency is <10ms.

### CODE REVIEW

**Comment by jason-dai** [comment]
LGTM

**Comment by hkvision** [comment]
jenkins passed: http://10.239.47.211:18888/job/ZOO-PR-Validation/2243/

File: zoo/src/main/scala/com/intel/analytics/zoo/models/image/imageclassification/ImageClassificationConfig.scala [hunk 1]
```diff
@@ -137,7 +136,7 @@ object ImagenetConfig {
   // Preprocessor for ResNet50 pre-trained in BigDL
   def bigdlResNetPreprocessor(): Preprocessing[ImageFeature, ImageFeature] = {
       ImageRandomResize(256, 256) ->
```
Comment (jason-dai) [inline zoo/src/main/scala/com/intel/analytics/zoo/models/image/imageclassification/ImageClassificationConfig.scala]: Can we just use resize?

Comment (hkvision) [reply zoo/src/main/scala/com/intel/analytics/zoo/models/image/imageclassification/ImageClassificationConfig.scala]: Tested that using resize will get almost compatible results and we can change to that. But for performance probably they are more or less the same (I will test further).

**Pull request merged**